scenes/MosaikHeer.scn

```diff
@@ -12,15 +12,13 @@
 /config/talk/A  +2.3 OFF ON %000000001111111111
 /config/talk/B -11.0 OFF ON %010000000000000000
 /config/osc -41.0 100.2 1k00 F1 PINK 18
-/config/userrout/out 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
```

Comment:
I think "user routing" was introduced with the newest M32-Edit software which I used to update the scene file - should have no influence on Sunday setup